docs/source/python_spec.rst

```diff
@@ -48,6 +50,14 @@ producer; the producer must synchronize or wait on the stream when necessary.
 In the common case of the default stream being used, synchronization will be
 unnecessary so asynchronous execution is enabled.
 
+A DLPack version can be requested by passing the ``version`` keyword. The
+consumer should call the ``__dlpack_info__`` method to get the maximum
+DLPack version supported by the producer and request for a version both
+support e.g. ``min(producer_version, consumer_version)``. If the consumer
+doesn't support any version below the producer's maximum version, a
+``BufferError`` should be raised. Similarly, If the producer doesn't
+support the requested version, it should raise a ``BufferError``.
+
```

Comment:
What happens if the consumer does not specify a version? As I understand things, for the forseeable future the producer should return a V1 structure. So effectively the default is `1`.

Reply:
> What happens if the consumer does not specify a version? As I understand things, for the forseeable future the producer should return a V1 structure. So effectively the default is `1`.

Yes, the default is 1. I will update the spec to say that instead.